ci-operator/step-registry/gather-network-trace/gather-network-trace-commands.sh

```diff
@@ -0,0 +1,141 @@
+then
+	echo "No kubeconfig, so no point in trying to generate a trace."
+	exit 0
+fi
+
+echo "Gathering network ovn artifacts and creating a trace ..."
+
+DIR=${ARTIFACT_DIR}/network-ovn-trace
+mkdir -p "${DIR}"
+
+# gather the masters-journal and workers-journal
+queue "${DIR}"/masters-journal oc adm node-logs --role=master --unify=false
+queue "${DIR}"/masters-journal-previous oc adm node-logs --boot=-1 --role=master --unify=false
+queue "${DIR}"/workers-journal oc adm node-logs --role=worker --unify=false
+queue "${DIR}"/workers-journal-previous oc adm node-logs --boot=-1 --role=worker --unify=false
+
+# gather the logfiles
+oc --request-timeout=5s get pods -n openshift-ovn-kubernetes \
+	--template '{{ range .items }}{{ $name := .metadata.name }}{{ $ns := .metadata.namespace }}{{ $node := .spec.nodeName }}{{ range .spec.containers }}{{ $node }}    {{ $name }}  -c {{ .name }}{{ "\n" }}{{ end }}{{ range .spec.initContainers }}{{ $node }} {{ $name }} -c {{ .name }}{{ "\n" }}{{ end }}{{ end }}' > /tmp/containers
+
+while IFS= read -r i; do
+  file=$( echo "${i}" | cut -d ' ' -f 2,4 | tr -s ' ' '_' )
+  con=$( echo "${i}" | cut -d ' ' -f 2,3,4 )
+  queue "${DIR}"/"${file}".log oc logs -n openshift-ovn-kubernetes --request-timeout=20s "${con}"
+  queue "${DIR}"/"${file}"_previous.log oc -n openshift-ovn-kubernetes logs --request-timeout=20s -p "${con}"
+done < /tmp/containers
+
+wait
+
+echo "Adding prefix ..."
+
+while IFS= read -r i; do
+  file=$( echo "${i}" | cut -d ' ' -f 2,4 | tr -s ' ' '_' )
+#  if [[ ! -f "${file}".log || ! -f "${file}"_previous.log ]] ; then
+#    continue
+#  fi
+  prefix=$( echo "${i}" | cut -d ' ' -f 1,2,4 )
+  container=$( echo "${i}" | cut -d ' ' -f 4 )
+  echo "file ${file}"
+  echo "prefix ${prefix}"
+  echo "container ${container}"
+  if [[ ${container} == "ovnkube-node" || ${container} == "ovnkube-master" ]] ; then
+	  # convert the timestamp to a standard format and add prefix
+	  grep -e "^I" -e "^E" -e "^W" -e "^N" "${DIR}"/"${file}".log | sed 's/ /-/;s/^[IEWDN]/& /' | \
+		  gawk '{ print $2  " | "  $0; }' | sed "s/|/ ${prefix} | /" > "${DIR}"/"${file}".log.pre
+	  grep -e "^I" -e "^E" -e "^W" -e "^N" "${DIR}"/"${file}"_previous.log | sed 's/ /-/;s/^[IEWDN]/& /' | \
+		  gawk '{ print $2  " | "  $0; }' | sed "s/|/ ${prefix} | /" > "${DIR}"/"${file}"_previous.log.pre
+  else
+	  # convert the timestamp to a standard format and add prefix
+	  sed 's/2020-//;s/-//;s/T/-/;s/Z//' "${DIR}"/"${file}".log | sed "s/|/ ${prefix} | /" > "${DIR}"/"${file}".log.pre
+	  sed 's/2020-//;s/-//;s/T/-/;s/Z//' "${DIR}"/"${file}"_previous.log | sed "s/|/ ${prefix} | /" > "${DIR}"/"${file}"_previous.log.pre
+  fi
+done < /tmp/containers
+echo "PHIL rm logs"
+#rm "${DIR}"/*log
+
+echo "Log files processed, on to journals"
+ls -l "${DIR}"/*journal*
+
+# prefix the journals and extract useful lines
+grep "hyperkube" "${DIR}"/masters-journal | grep -e "kubelet.go" -e "event.go" > "${DIR}"/journal
+wc "${DIR}"/journal
+echo "PHIL 1"
```

Comment:
@pecameron left-over debugging?

Reply:
@dcbw this script will be replaced with a call to oc adm must-gather when 
https://github.com/openshift/must-gather/pull/156
lands. PTAL must-gather/pull/156